src/main/java/io/cdap/e2e/utils/ConstantsUtil.java

```diff
@@ -42,6 +43,7 @@
   public static final String ERROR_MSG_MANDATORY = "Required property 'PROPERTY' has no value.";
   public static final String DEFAULT_PLUGIN_PROPERTIES_FILE = "pluginParameters.properties";
   public static final String DEFAULT_ERROR_PROPERTIES_FILE = "errorMessage.properties";
+  public static final String DEFAULT_DATACY_ATTRIBUTES_FILE = "pluginDataCyAttributes.properties";
```

Comment:
Why do we need this file?

Reply:
With Common steps introduced in this PR, we will be passing the data-cy attribute of a property while configuring the plugin. Same data-cy attribute will be passed in all the tests. We thought of adding this property file with the data-cy mapping so we will only need to make a change in one place in case there is a change in the data-cy attribute of a property.